ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -2062,6 +2071,66 @@ - (ORKAnswerFormat *)impliedAnswerFormat {
 @end
 
 
+#pragma mark - ORKConfirmTextAnswerFormat
+
+@implementation ORKConfirmTextAnswerFormat
+
+- (Class)questionResultClass {
+    return [ORKBooleanQuestionResult class];
+}
+
+- (instancetype)initWithOriginalItemIdentifier:(NSString *)originalItemIdentifier {
+    self = [super init];
+    if (self) {
+        _originalItemIdentifier = [originalItemIdentifier copy];
```

Comment:
If `_originalItemIdentifier` is only set through init and `readonly`, then we need to make sure it is not nil at here, instand of in `validateParameters`.